src/components/App/App.vue

```diff
@@ -32,35 +33,52 @@ export default {
   data: () => ({
     locations: [],
     cities: [],
-    states: []
+    states: [],
+    dropdownErrorMessage: ''
   }),
   mounted: function () {
     this.$nextTick(function () {
       this.retrieveLocalStorage()
-      fetchAPI.getStates().then(data => {
-        this.states = data.data
-      })
+      fetchAPI.getStates()
+        .then(data => {
+          this.states = data.data
+        })
+        .catch(() => {
+          this.dropdownErrorMessage = 'Please wait a minute and refresh the page. Something went wrong with the server.'
```

Comment:
Wonder if we could DRY up this message somehow since it's currently the same in ~4 different methods?